.changeset/little-jeans-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/overlays': patch
```

Comment:
Patch instead of minor
 because those methods already existed?

Reply:
because semver < 1.x.x and minor would therefore be a breaking change... 

but it doesn't really matter as I think there is already a minor change for overlays lined up for the next release... jup will be 0.20.0 https://github.com/ing-bank/lion/pull/936

so can do minor or patch here... both is fine with me